app/views/providers/allocations/edit.html.erb

```diff
@@ -12,6 +12,7 @@
                     id: @allocation.id
                   ),
                   model: @allocation,
+                  scope: "",
```

Comment:
What is the rationale behind adding an empty string as a scope?

Reply:
The reason scope is an empty string is that if scope if not defined then it uses the model class name as a namespace for the various fields that the from builder renders. Normally this shouldn't be a problem but because @allocation could be either a InitialRequestForm or RepeatRequestFrom class and potentially others where we reuse the markup it becomes difficult to use  one single namespace. So in this instance we clear the namespace with an empty string.